web/modules/custom/joinup_core/src/GroupHelper.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\joinup_core;
+
+use Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException;
+use Drupal\Component\Plugin\Exception\PluginNotFoundException;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\rdf_entity\Entity\Rdf;
+use Drupal\rdf_entity\RdfEntitySparqlStorageInterface;
+use Drupal\rdf_entity\RdfInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Helper methods for working with groups (collections and solutions).
+ */
+class GroupHelper implements GroupHelperInterface {
```

Comment:
Oh you're right, this is left over from an earlier iteration and is not being used any more.

Reply:
I was about to remove this but it could be helpful for future code, and also to collect further code that is acting on both collections and solutions. Are you OK to keep it or rather remove it?

Reply:
Nahh I prefer to not keep unused code in the project. It is not like the project is not huge already :)